Libraries/Components/HyperlinkWindows/HyperlinkWindows.windows.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @providesModule HyperlinkWindows
+ */
+'use strict';
+
+var PropTypes = require('prop-types');
+var React = require('React');
+var ReactNativeViewAttributes = require('ReactNativeViewAttributes');
+var ViewPropTypes = require('ViewPropTypes');
+var requireNativeComponent = require('requireNativeComponent');
+
+const ColorPropType = require('ColorPropType');
+const EdgeInsetsPropType = require('EdgeInsetsPropType');
+const StyleSheetPropType = require('StyleSheetPropType');
+const Text = require('Text');
+const TextStylePropTypes = require('TextStylePropTypes');
+const TouchableWithoutFeedback = require('TouchableWithoutFeedback');
+
+const stylePropType = StyleSheetPropType(TextStylePropTypes);
+
+class HyperlinkWindows extends React.Component {
+  props: {
+    ellipsizeMode: 'head' | 'middle' | 'tail' | 'clip',
+    numberOfLines: number,
+    textBreakStrategy: 'simple' | 'highQuality' | 'balanced',
+    onLayout: Function,
+    onPress: Function,
+    onLongPress: Function,
+    pressRetentionOffset: EdgeInsetsPropType,
+    selectable: boolean,
+    selectionColor: ColorPropType,
+    suppressHighlighting: boolean,
+    style: stylePropType,
+    testID: string,
+    nativeID: string,
+    allowFontScaling: boolean,
+    accessible: boolean,
+    adjustsFontSizeToFit: boolean,
+    minimumFontScale: number,
+    disabled: boolean,
+  };
+
+  static propTypes = {
+    ...Text.PropTypes,
+  };
+
+  static defaultProps = {
+  };
+
+  render() {
+    return (
+        <TouchableWithoutFeedback 
```

Comment:
>TouchableWithoutFeedback [](start = 9, length = 24)

Why do you need a touchable, the xaml hyperlink seems to support clicking already!

Reply:
Agree - should we use the `Touchable` infrastructure, or the built-in [`Click`](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.documents.hyperlink.click) event for Hyperlink?

---
In reply to: [182928994](https://github.com/Microsoft/react-native-windows/pull/1794#discussion_r182928994) [](ancestors = 182928994)

Reply:
For the long press. The Hyperlink control doesn't allow for getting the mouse down/up events necessary to detect long press.